src/index.jsx

```diff
@@ -134,7 +134,7 @@ export class Recogito {
 
   getAnnotations = () => {
     const annotations = this._app.current.getAnnotations();
-    return annotations.map(a => a.underlying);
+    return annotations.map(a => ({...a.underlying, creator: a.creator}));
```

Comment:
Accessing `creator` is also required through a change on client-core. Otherwise this will always return undefined

Reply:
Passed into underlying instead of opts